modules/odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assembly:
+//
+// * web_console/customizing-the-web-console.adoc
+
+:_content-type: PROCEDURE
+
+[id="odc_customizing-a-developer-catalog-or-its-sub-catalogs-using-the-form-view_{context}"]
+= Customizing a Developer catalog or it's sub-catalogs using the form view
+
+You can use a similar procedure to customize the following UI items:
+** Quick starts visible to users
+** Cluster roles accessible to a project
+** Actions visible on the +Add page
+
+.Prerequisites
+
+* You must have administrator privileges.
+
+.Procedure
+
+. In the *Administrator* perspective, navigate to *Administration* -> *Cluster Settings*.
```

Comment:
Where can I find this perspective?

Reply:
This is displayed by default on the Web UI. You can select either this or Developer as per my knowledge.

Reply:
Got it! thanks